### ISSUE TITLE
WCAG 4.1.2 tests 9 and 14 possible false positives

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#75]
[This PDF document](https://github.com/veraPDF/veraPDF-wcag-algs/files/6969738/test-document-10.pdf) generates the following errors that appear to be false positives:

- **WCAG 4.1.2 test 9 "Missing table elements"** points to text that obviously appears to be a real table. It is probably also tagged as such because Swink indicated that this is a well tagged file. It's not clear why the algorithm thinks this is not a table.
- **WCAG 4.1.2 test 15 "Paragraph is incorrectly tagged as a numbered heading"** points to two occurrences. At least the second (the line "1. Bezoekers") is obviously a real heading. What is even more curious is that the algorithm does not give this error for other, similar headings in the document such as "2. Bezoeken", "3. Acties", etc. So the algorithm does not only give a false positive, it also behaves inconsistently.

**Comment by bdoubrov** [comment]
The issue with false "Missing table elements" is fixed. 

As fir the incorrect numbered heading, the problem is that both H2 (Definities) and H3 (1. Bezoekers) on page 2 have identical font style. This is why they are not recognized as headings but rather usual paragraphs.

We can enhance the heuristics of heading detection to take into account, for example, leading number in a one-line paragraph.

**Comment by bdoubrov** [comment]
We'll implement two changes in heading recognition:
- increase (slightly) probability of a heading if we see that the author tagged it as a heading
- check the leading numbering of a line

**Comment by bdoubrov** [comment]
Second item is implemented in https://github.com/veraPDF/veraPDF-wcag-algs/pull/105

**Comment by jzuidweg** [comment]
This issue is marked as fixed-in-dev but **WCAG 4.1.2 test 9** "_Missing table elements_" continues giving false positives. I'm not closing this issue yet.

**Comment by jzuidweg** [comment]
Closing this issue, but opening new issue #273 for recognition of headers.
